### PR TITLE
Update readme with details on coronavirus and transition pages.

### PR DESCRIPTION
**Pull request by hannako**
## What

Collections might be getting some on call action over the Christmas and New Years eve periods, so amending the readme with details of Coronavirus and Transition pages. 

[Rendered readme](https://github.com/alphagov/collections/blob/69b9ca76d344e1292ac0c67478a3a06cbba79724/README.md) 👈 

[Rendered changes](https://github.com/alphagov/collections/pull/2160/commits/69b9ca76d344e1292ac0c67478a3a06cbba79724?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) 👈 

---
[Trello](https://trello.com/c/A0cz6Ir4/688-review-developer-docs)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,32 +99,39 @@ For example:
...
 
 This is a public facing Ruby on Rails application that retrieves browse content from APIs and presents it.
-There is no underlying persistence layer and all content is retrieved from external sources.
+There is no database, and for the majority of pages it serves, collections reads content from external sources.
+
+### Transition landing page ([ /transition ](https://www.gov.uk/transition))
+All content for the transition landing pages are currently read from yaml files. [Welsh](config/locales/cy/coronavirus_landing_page.yml) and [English](config/locales/en/coronavirus_landing_page.yml) translations are available.
+
+### DIT landing page ([ /eubusiness ](https://www.gov.uk/eubusiness))
+All content for the DIT landing pages are currently read from yaml files. [English](config/locales/en/dit_landing_page.yml), [German](config/locales/de/dit_landing_page.yml), [Spanish](config/locales/es/dit_landing_page.yml), [French](config/locales/fr/dit_landing_page.yml), [Italian](config/locales/it/dit_landing_page.yml), [Dutch](config/locales/nl/dit_landing_page.yml) and [Polish](config/locales/pl/dit_landing_page.yml) translations are available.
```
Comment (benthorner) [inline README.md:108]:
```suggestion
All content for the DIT landing pages are currently read from yaml files e.g. for [English](config/locales/en/dit_landing_page.yml). See the [content item's available translations](https://www.gov.uk/api/content/eubusiness) for the current list.
```

This should be slightly more robust for the future 🤞.

**Review by benthorner — COMMENTED** [review]
Looks good to me - I'm not too familiar with Collections.

I've made a couple of suggestions. I'm also wondering if there's a way we could make a clear distinction between what collections is _meant_ to do (the original README), and stuff that's been jammed in to it and isn't particularly related to its purpose. This should also make it easier to identify for removal in the future.

What do you think?

File: README.md [hunk 2]
```diff
@@ -95,32 +99,39 @@ For example:
 ## Technical documentation
 
 This is a public facing Ruby on Rails application that retrieves browse content from APIs and presents it.
-There is no underlying persistence layer and all content is retrieved from external sources.
+There is no database, and for the majority of pages it serves, collections reads content from external sources.
+
+### Transition landing page ([ /transition ](https://www.gov.uk/transition))
+All content for the transition landing pages are currently read from yaml files. [Welsh](config/locales/cy/coronavirus_landing_page.yml) and [English](config/locales/en/coronavirus_landing_page.yml) translations are available.
```
[the remark this answers is not in the document]
Comment (hannako) [reply README.md:105]: woops! thanks

File: README.md (hunk 1 above)
Comment (hannako) [reply README.md:108]: 👍

File: README.md [hunk 3]
```diff
@@ -91,36 +97,50 @@ For example:
...
+Collections currently renders the following campaign pages:
+
+#### Transition landing page ([ /transition ](https://www.gov.uk/transition))
+All content for the transition landing pages are currently read from yaml files. [Welsh](config/locales/cy/transition_landing_page.yml) and [English](config/locales/en/transition_landing_page.yml) translations are available.
+
+#### DIT landing page ([ /eubusiness ](https://www.gov.uk/eubusiness))
+All content for the DIT landing pages are currently read from yaml files e.g. for [English](config/locales/en/dit_landing_page.yml). See the [content item's available translations](https://www.gov.uk/api/content/eubusiness) for the current list.
+
+#### Coronavirus pages:
+  - Landing page ( [gov.uk/coronavirus](https://www.gov.uk/coronavirus) ),
```
Comment (benthorner) [inline README.md:111]:
```suggestion
  - Landing page ( [gov.uk/coronavirus](https://www.gov.uk/coronavirus) )
```

**Review by benthorner — APPROVED** [review]
Nice! For me, it's also clearer seeing the documentation about campaign pages separated out. Thanks for following up on that suggestion 💯.

**Pull request merged**